packages/table/test/cellTests.tsx

```diff
@@ -33,7 +33,7 @@ describe("Cell", () => {
     it("renders loading state", () => {
         const cellHarness = harness.mount(<Cell loading={true} />);
         const cellElement = cellHarness.element.children[0];
-        expect(cellElement.classList.contains(Classes.LOADING)).to.be.true;
+        expect(cellElement.classList.contains("pt-loading")).to.be.true;
```

Comment:
wait this is worse than on the left cuz you're not even using the constant. check for `.pt-skeleton`.